rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/component/ComponentController.java

```diff
@@ -104,6 +107,25 @@
         return new ResponseEntity<>(userHalResource, HttpStatus.OK);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @RequestMapping(value = COMPONENTS_URL + "/{ids}", method = RequestMethod.DELETE)
+    public ResponseEntity<List<BatchOperationResult>> deleteComponents(
+            @PathVariable("ids") List<String> idsToDelete, OAuth2Authentication oAuth2Authentication) throws TException {
+        User user = restControllerHelper.getSw360UserFromAuthentication(oAuth2Authentication);
+        List<BatchOperationResult> results = new ArrayList<>();
+        for(String id:idsToDelete) {
+            RequestStatus requestStatus = componentService.deleteComponent(id, user);
+            HttpStatus status;
+            if(requestStatus == RequestStatus.SUCCESS) {
+                status = HttpStatus.OK;
+            } else {
+                status = HttpStatus.INTERNAL_SERVER_ERROR;
```

Comment:
If a component contains releases (release.size >0) the method call `componentService.deleteComponent`
will return `RequestStatus.IN_USE`. 
Instead of `500 - internal server error` we could throw `409 conflict`, what do you think?

Reply:
I'm now using `409 conflict` in this case